features/0066-non-repudiable-cryptographic-envelope/README.md

```diff
@@ -0,0 +1,251 @@
+# Non-Repudiable Signature for Cryptographic Envelope
+- Author: Kyle Den Hartog (indy@kyledenhartog.com)
+- Start Date: 04/15/2019
+
+## Status
+- Status: [![proposed](/proposed.png)](/README.md#hipe-lifecycle)
```

Comment:
We've been keeping the status in ALL CAPS to make it more visible against the rest of the text. Minor quibble only.

Reply:
Shouldn't the hyperlink be to #rfc-lifecycle?